From 678b233f08ed9188b06ae7e841d6e680f6207e36 Mon Sep 17 00:00:00 2001
From: "A.I" <ailis@paw.zone>
Date: Tue, 8 May 2018 18:00:12 -0700
Subject: [PATCH] Fix checking NO_NEW_PRIVS for cross compilation

Changed to AC_COMPILE_IFELSE as AC_RUN_IFELSE cannot be run while cross compiling.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index db4465d0..efe81b7f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -150,7 +150,7 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#define _GNU_SOURCE
                   )
 
 AC_MSG_CHECKING([for feature: NO_NEW_PRIVS])
-AC_RUN_IFELSE([AC_LANG_PROGRAM([[
+AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
                                      #include <sys/prctl.h>
                                      #ifndef PR_SET_NO_NEW_PRIVS
                                      # define PR_SET_NO_NEW_PRIVS 38
-- 
2.14.1

